### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by Sixtease** [deepakbaby/se_relativisticgan#5]
Would you be so kind to specify the package versions needed? Especially which Tensorflow version is required and what CUDA/CuDNN versions you had success using.

**Comment by deepakbaby** [comment]
I unfortunately dont have that information now. I will check if it runs fine in my current environment. It was defenitely tensorflow < 2.0. Have you tried running the code in your current environment?

**Comment by Sixtease** [comment]
I have. I am trying now with tensorflow-gpu==1.14.0 and it seems to run with just some deprecation warnings.

**Comment by deepakbaby** [comment]
Keep an eye on the adversarial losses. After a few iterations, they all should stay around  0.69 (which is `- log 0.5`) to be in the equilibrium. If the loss fluctuates too much even after a few tens of epochs, then something is wrong.

**Comment by deepakbaby** [comment]
Hi,
My package versions were
Keras v2.3.1
Tensorflow v1.13.2
